firmware/shared/os/event_flags.h

```diff
@@ -0,0 +1,17 @@
+/// @author Matteo Tullo
+/// @date 2024-02-24
+
+#pragma once
+
+#include <cstdint>
+#include "shared/util/os.h"
+
+namespace shared::osDataType {
```

Comment:
Namespace name does not follow style guide

Reply:
As per discussion, removed osDataType and converted all RTOS related classes and functions to use the `::os` namespace